cmake/IgnCodeCoverage.cmake

```
@@ -132,7 +139,7 @@ FUNCTION(ign_setup_target_for_coverage)
     # Remove negative counts
     COMMAND sed -i '/,-/d' ${_outputname}.info
     COMMAND ${LCOV_PATH} ${_branch_flags} -q
-      --remove ${_outputname}.info '*/test/*' '/usr/*' '*_TEST*' '*.cxx' 'moc_*.cpp' 'qrc_*.cpp' '*.pb.*' --output-file ${_outputname}.info.cleaned
+      --remove ${_outputname}.info '*/test/*' '/usr/*' '*_TEST*' '*.cxx' 'moc_*.cpp' 'qrc_*.cpp' '*.pb.*' '*/build/*' '*/install/*' '*/opt/ros/*' ${IGNORE_LIST} --output-file ${_outputname}.info.cleaned
```

When does `/opt/ros/` need to be ignored? Did it appear in a particular repo?

---

Yes. In the local coverage report for `gz-physics`

---

I tried the below suggestion and it does not appear in my local coverage report for `gz-physics`. This should be removed since `ros` is separate from gazebo
```suggestion
      --remove ${_outputname}.info '*/test/*' '/usr/*' '*_TEST*' '*.cxx' 'moc_*.cpp' 'qrc_*.cpp' '*.pb.*' '*/build/*' '*/install/*' ${IGNORE_LIST} --output-file ${_outputname}.info.cleaned
```